openmetadata-ui/src/main/resources/ui/src/utils/DataInsightUtils.tsx

```diff
@@ -20,11 +21,13 @@ import {
   last,
   toNumber,
 } from 'lodash';
+import moment from 'moment';
```

Comment:
aren't we considering removing the moment?

Reply:
Antd has a dependency of moment, so we have to use moment for antd date picker.

Reply:
@chirag-madlani Any suggestion over here?

Reply:
No @Sachin-chaurasiya that's not correct. Let's connect to resolve this

Reply:
Sure @chirag-madlani

Reply:
We have tried different ways to eliminate the moment but have had no luck so we will take care of eliminating the moment later.